### PR TITLE
fix: add overflow to card content (for card/help)

### PR DESCRIPTION
**Pull request by charlielizzy**

### CODE REVIEW

File: src/components/organisms/Card/CardContent/CardContent.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ const CardContent = styled.div`
   flex: 1;
   display: flex;
   flex-direction: column;
+  overflow-wrap: break-word;
```
Comment (OlenaKashuba) [inline src/components/organisms/Card/CardContent/CardContent.tsx:9]: true CSS power in action - and a fix of WCAG AA violation!

**Review by OlenaKashuba — APPROVED** [review]
Love this

**Review by admmasters — APPROVED** [review]

**Pull request merged**